### ISSUE TITLE
Verify code input has funny interface

### ISSUE THREAD

**Issue by mikedotexe** [near/near-wallet#613]
## Story

As a user, I want to create an account with Wallet and, when entering my verify code, see form elements that are straightforward and what I'm used to, given my experience of writing in verify codes because I use 2FA everywhere like a good employee.

## Acceptance Criteria

Form elements that take an input to be typed do not have extra widgets that cause me a moment of confusion.

![email-verify-code](https://user-images.githubusercontent.com/1042667/83078378-55676280-a02e-11ea-96fc-8e6b8b469335.gif)

Basically, I'd prefer to hide the little plus/minus buttons on this element. It probably gives "free validation" on numbers, but was an odd feeling when I saw it.
